min/lib/CSSmin.php

```diff
@@ -765,7 +765,9 @@ private function str_slice($str, $start = 0, $end = FALSE)
     private function normalize_int($size)
     {
         if (is_string($size)) {
-            switch (substr($size, -1)) {
+            $letter = substr($size, -1);
+            $size = intval($size);
```

Comment:
the `int` cast is preferable to calling the `intval` function as it's faster, and does exactly the same thing/